File: app/src/main/java/com/babestudios/companieshouse/data/model/officers/appointments/Appointment.java
package com.babestudios.companieshouse.data.model.officers.appointments;


import com.babestudios.companieshouse.data.model.officers.Address;
import com.babestudios.companieshouse.data.model.officers.Identification;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Appointment {

	@SerializedName("address")
	public Address address;
	@SerializedName("appointed_before")
	public String appointedBefore;
	@SerializedName("appointed_on")
	public String appointedOn;
	@SerializedName("appointed_to")
	public AppointedTo appointedTo;
	@SerializedName("country_of_residence")
	public String countryOfResidence;
	@SerializedName("former_names")
	public List<FormerName> formerNames = null;
	@SerializedName("identification")
	public Identification identification;
	@SerializedName("is_pre_1992_appointment")
	public String isPre1992Appointment;
	@SerializedName("links")
	public Links links;
	@SerializedName("name")
	public String name;
	@SerializedName("name_elements")
	public NameElements nameElements;
	@SerializedName("nationality")
	public String nationality;
	@SerializedName("occupation")
	public String occupation;
	@SerializedName("officer_role")
	public String officerRole;
	@SerializedName("resigned_on")
	public String resignedOn;

}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/officers/appointments/Appointment.kt
package com.babestudios.companyinfouk.data.model.officers.appointments


import com.babestudios.companyinfouk.data.model.officers.Address
import com.babestudios.companyinfouk.data.model.officers.Identification
import com.google.gson.annotations.SerializedName

class Appointment {

	@SerializedName("address")
	var address: Address? = null
	@SerializedName("appointed_before")
	var appointedBefore: String? = null
	@SerializedName("appointed_on")
	var appointedOn: String? = null
	@SerializedName("appointed_to")
	var appointedTo: AppointedTo? = null
	@SerializedName("country_of_residence")
	var countryOfResidence: String? = null
	@SerializedName("former_names")
	var formerNames: List<FormerName>? = null
	@SerializedName("identification")
	var identification: Identification? = null
	@SerializedName("is_pre_1992_appointment")
	var isPre1992Appointment: String? = null
	@SerializedName("links")
	var links: Links? = null
	@SerializedName("name")
	var name: String? = null
	@SerializedName("name_elements")
	var nameElements: NameElements? = null
	@SerializedName("nationality")
	var nationality: String? = null
	@SerializedName("occupation")
	var occupation: String? = null
	@SerializedName("officer_role")
	var officerRole: String? = null
	@SerializedName("resigned_on")
	var resignedOn: String? = null

}
